### ISSUE TITLE
Keyboard navigation with component and embedded types

### ISSUE THREAD

**Issue by ArtemAvramenko** [uxsolutions/bootstrap-datepicker#2425]
How to enable keyboard navigation in component or embedded mode?

**Comment by havok89** [comment]
Having an issue with this too.

on the sandbox demo you can see that if you select component type and trigger the datepicker with the button you cannot use keyboard navigation on it but if you trigger the datepicker with the input box you can.

triggering with the input box then closing datepicker and triggering with the button still allows you to keyboard navigate but just the button on its own doesnt
